testing/vault/vault.initd

```diff
@@ -10,9 +10,10 @@ pidfile="/run/${SVCNAME}.pid"
 
 start_pre() {
 	checkpath -f -m 0644 -o ${SVCNAME}:${SVCNAME} "$VAULT_LOG_FILE"
+	setcap cap_ipc_lock=+ep $(readlink -f $(which vault))
```

Comment:
@jirutka fixed. Sorry didn't know I could to setcap in APKBUILD just like install.